src/helpers/assets.ts

```diff
@@ -414,7 +415,7 @@ export const buildBriefUniqueTokenList = (
 
     result.push({ type: 'NFT_SPACE_AFTER', uid: `${family}-space-after` });
   }
-  if (hiddenUniqueTokensIds.length > 0) {
+  if (hiddenUniqueTokensIds.length > 0 && listType === 'wallet') {
```

Comment:
fyi, the other list types are `"ens-profile"` & `"select-nft"`, where we do not want to show hidden NFTs.

Reply:
yesh